lumibot/brokers/schwab.py

```
@@ -1682,6 +1817,144 @@ def _parse_simple_order(self, schwab_order: dict, strategy_name: str) -> List[Or
+                multiplier=getattr(getattr(stored_order, "asset", None), "multiplier", 1) or 1,
+            )
+            self._log_schwab_lifecycle_event(
+                "order.lifecycle.callback",
+                stored_order,
+                callback=event,
+                broker_status=observed_status,
+                local_status=stored_order.status,
+            )
+            return
+
+        if Order.is_equivalent_status(observed_status, self.CANCELED_ORDER):
```

Pylint (4.0.7)

[error] Class 'Order' has no 'is_equivalent_status' member

(E1101)